include/lsst/meas/base/PixelFlags.h

```diff
@@ -74,8 +80,18 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
         CR,
         CR_CENTER,
         BAD,
+        CLIPPED,
```

Comment:
Given that you're not using `FlagHandler`, we can remove this enum entirely rather than add to it.